### PR TITLE
updated fastlane configuration to support match on dev machines

### PR DESCRIPTION
**Pull request by DanielSinclair**
Improves defaults for `bundle exec fastlane match` to reduce dev machine setup time

## What changed (plus any additional context for devs)
Fastlane configuration files:
- created a Matchfile with dev environment defaults
- added other bundle identifiers to Appfile
- added team_name to Appfile (for multi-org dev accounts)
- removed Match duplication in Fastfile

## Dev checklist for QA: what to test
- Bitrise CI builds and deployment

## Final checklist
- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Comment by brunobar79** [comment]
Also, would you mind manually trigger the testflight flow on bitrise and link the succeeding build in the PR description? Thanks!

**Comment by brunobar79** [comment]
> 

I guess it's because it's still a draft

**Review by brunobar79 — APPROVED** [review]

File: ios/fastlane/Fastfile [hunk 1]
```diff
@@ -57,9 +57,7 @@ platform :ios do
 
     match(
       force: true,
```
Comment (brunobar79) [inline ios/fastlane/Fastfile:59]:
```suggestion
```

**Review by brunobar79 — APPROVED** [review]

**Review by estrattonbailey — APPROVED** [review]
Any reason not to merge this one? I re-ran the e2e, maybe it'll pass now 😛

**Review by skylarbarrera — APPROVED** [review]
lgtm

**Comment by jkadamczyk** [comment]
I will try running a testflight build to see if it works. If it does we are clear to merge it I think

**Pull request closed without merging**